### PR TITLE
gweled: init at unstable-2018-02-15

### PR DESCRIPTION
**Pull request by bignaux**
###### Motivation for this change

The latest released is 2011-03-31 and won't work. This is why i packed unstable. 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/games/gweled/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  nativeBuildInputs = [ wrapGAppsHook intltool autoreconfHook pkgconfig ];
+
+  buildInputs = [ gtk2 libmikmod librsvg hicolor-icon-theme libcanberra-gtk2 ];
+
+  configureFlags = [ "--disable-setgid" ];
+
+  meta = with stdenv.lib; {
+    description = "Bejeweled clone game";
+    homepage = https://gweled.org/;
```
Comment (JohnAZoidberg) [inline pkgs/games/gweled/default.nix]: Please use quotes around URLs. We might remove unquoted syntax: https://github.com/NixOS/rfcs/pull/45

**Comment by JohnAZoidberg** [comment]
Works. I played for a few minutes :D

**Comment by bignaux** [comment]
best game is to package it for nixos ;)

**Comment by timokau** [comment]
What do you mean by "won't work"?

**Pull request merged**